File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/domain/Models/ConferenceInfo.cs
using System;

namespace ConferenceManagement
{
    public class ConferenceInfo
    {
        public string AccessCode { get; private set; }
        public ConferenceOwner Owner { get; private set; }
        public string Slug { get; private set; }
        public string Name { get; private set; }
        public string Description { get; private set; }
        public string Location { get; private set; }
        public string Tagline { get; private set; }
        public string TwitterSearch { get; private set; }
        public DateTime StartDate { get; private set; }
        public DateTime EndDate { get; private set; }

        public ConferenceInfo() { }
        public ConferenceInfo(string accessCode, ConferenceOwner owner, string slug, string name, string description, string location, string tagline, string twitterSearch, DateTime startDate, DateTime endDate)
        {
            AccessCode = accessCode;
            Owner = owner;
            Slug = slug;
            Name = name;
            Description = description;
            Location = location;
            Tagline = tagline;
            TwitterSearch = twitterSearch;
            StartDate = startDate;
            EndDate = endDate;
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/models/ConferenceInfo.java
package com.microsoft.conference.management.domain.models;

import java.util.Date;

public class ConferenceInfo {
    public String accessCode;
    public ConferenceOwner owner;
    public String slug;
    public String name;
    public String description;
    public String location;
    public String tagline;
    public String twitterSearch;
    public Date startDate;
    public Date endDate;

    public ConferenceInfo() {
    }

    public ConferenceInfo(String accessCode, ConferenceOwner owner, String slug, String name, String description, String location, String tagline, String twitterSearch, Date startDate, Date endDate) {
        this.accessCode = accessCode;
        this.owner = owner;
        this.slug = slug;
        this.name = name;
        this.description = description;
        this.location = location;
        this.tagline = tagline;
        this.twitterSearch = twitterSearch;
        this.startDate = startDate;
        this.endDate = endDate;
    }
}
